pkg/syncer/resource_syncer_test.go

```diff
@@ -957,6 +958,41 @@ func testListResourcesBySelector() {
 	})
 }
 
+func testRequeueResource() {
+	d := newTestDiver(test.LocalNamespace, "", syncer.LocalToRemote)
```

Comment:
nit

P.S: I know its an existing API and not added by this PR ;-) 

```suggestion
	d := newTestDriver(test.LocalNamespace, "", syncer.LocalToRemote)
```

Reply:
oh - never noticed that. I'll fix it.